### ISSUE TITLE
Fullscreen resets the server sreen size

### ISSUE THREAD

**Issue by macias** [FreeRDP/FreeRDP#2986]
Thank you for a great program. I am using its Windows build to connect in such setup:

* server -- Linux running VBox
* client -- Windows

I try to connect with fullscreen turned on ("/f") but it does not work -- instead fullscreen VBox running on server is reset to small size window, then wfreerdp uses this size as its own fullscreen resulting in pretty small window on client computer.

For comparison I tried other rdp clients and they do not reset server screen size and thus they are able to use the size set on the server.

Tested today on nightly build and also using version 1.1.

**Comment by bmiklautz** [comment]
@macias just to clarify you are connecting with wfreerdp to virtualbox (vrdp)?

**Comment by macias** [comment]
@bmiklautz, yes, exactly, this is the case. However small correction, the other client I tested (2X RDP) "cheated" -- I rechecked it and it also resets the screen size on server, but on client it runs fullscreen by scaling the display. So it is not true fullscreen really.

**Comment by bmiklautz** [comment]
@macias what do you have installed inside VBox? - Are the guest utils installed or not? Also the VBox version would be interesting.

**Comment by macias** [comment]
@bmiklautz , I have Windows 7 inside VBox. The guest additions are installed. I tested older VBox 4.3.x version, and then I installed the newest 5.0.10. Both versions behave the same. Just in case, host system for VBox is openSUSE 13.2.

**Comment by macias** [comment]
I cannot tell because for some time I don't use this setup anymore.
